sr_multi_moveit/sr_multi_moveit_config/launch/sensor_manager.launch.xml

```diff
@@ -1,15 +1,19 @@
 <launch>
+  <!-- 'octomap_date_type' options are 'pointcloud' or 'depthmap' -->
+  <arg name="octomap_date_type" default="pointcloud"/>
```

Comment:
```suggestion
  <arg name="octomap_data_type" default="pointcloud"/>
```

Reply:
Please change this in all relevant places

Reply:
Oh woww, seriously well spotted, thanks man

Reply:
cf2ce8a9d00699a352afe091c6bdf18b1472a08a